dotCMS/src/main/java/com/dotmarketing/portlets/rules/util/RulePermissionableUtil.java

```diff
@@ -37,9 +37,12 @@ public static Permissionable findParentPermissionable(String parent) throws DotD
                 if(iden.getAssetType().equals("folder")){
                     permissionableParent = APILocator.getFolderAPI().find(parent,systemUser,false);
                 }else{
-					Contentlet contentlet = APILocator.getContentletAPI().findContentletByIdentifier(parent, false,
-							APILocator.getLanguageAPI().getDefaultLanguage().getId(), systemUser, false);
-					if (contentlet.isHost()) {
+					//Contentlet contentlet = APILocator.getContentletAPI().findContentletByIdentifier(parent, false,
+					//		APILocator.getLanguageAPI().getDefaultLanguage().getId(), systemUser, false);
+                    Contentlet contentlet = APILocator.getContentletAPI()
+                            .search("+identifier:" + parent + " +working:true", 1, 0, null, systemUser, false)
```

Comment:
be aware you can get an index out of bounds exception, expecting a return value, might need to handle this exception

Reply:
the Exception catch is already in the code, below